compiler/crates/graphql-ir/src/node_identifier.rs

```diff
@@ -405,6 +409,16 @@ impl LocationAgnosticPartialEq for Argument {
     }
 }
 
+impl LocationAgnosticPartialEq for Option<&Argument> {
```

Comment:
This impl should be redundant from `Option<T>`, but I couldn't work out the right ref incantation